x-pack/plugins/security_solution/public/common/components/guided_onboarding/tour.tsx

```diff
@@ -0,0 +1,182 @@
+
+import type { EuiTourStepProps } from '@elastic/eui';
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTourStep,
+  EuiText,
+  EuiSpacer,
+  EuiImage,
+  useIsWithinBreakpoints,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { tourConfig } from './tour_config';
+
+const SECURITY_TOUR_ACTIVE_KEY = 'guidedOnboarding.security.tourActive';
+const SECURITY_TOUR_STEP_KEY = 'guidedOnboarding.security.tourStep';
+const getIsTourActiveFromLocalStorage = (): boolean => {
+  return Boolean(JSON.parse(String(localStorage.getItem(SECURITY_TOUR_ACTIVE_KEY))));
+};
+const saveIsTourActiveToLocalStorage = (isTourActive: boolean): void => {
+  localStorage.setItem(SECURITY_TOUR_ACTIVE_KEY, JSON.stringify(isTourActive));
+};
+
+const getTourStepFromLocalStorage = (): number => {
+  return Number(JSON.parse(String(localStorage.getItem(SECURITY_TOUR_STEP_KEY) ?? 1)));
+};
+const saveTourStepToLocalStorage = (step: number): void => {
+  localStorage.setItem(SECURITY_TOUR_STEP_KEY, JSON.stringify(step));
+};
+
+const minWidth: EuiTourStepProps['minWidth'] = 360;
+const maxWidth: EuiTourStepProps['maxWidth'] = 360;
+const offset: EuiTourStepProps['offset'] = 30;
+const repositionOnScroll: EuiTourStepProps['repositionOnScroll'] = true;
+
+const getSteps = (tourControls: {
+  activeStep: number;
+  incrementStep: () => void;
+  resetTour: () => void;
+}) => {
+  const { activeStep, incrementStep, resetTour } = tourControls;
+  const footerAction = (
+    <EuiFlexGroup alignItems="center">
+      <EuiFlexItem>
+        <EuiButtonEmpty
+          size="xs"
+          color="text"
+          onClick={() => resetTour()}
+          data-test-subj="onboarding--securityTourSkipButton"
+        >
+          <FormattedMessage
+            id="xpack.securitySolution.guided_onboarding.skipTour.buttonLabel"
+            defaultMessage="Skip tour"
+          />
+        </EuiButtonEmpty>
+      </EuiFlexItem>
+      <EuiFlexItem>
+        <EuiButton
+          size="s"
+          onClick={() => incrementStep()}
+          color="success"
+          data-test-subj="onboarding--securityTourNextStepButton"
+        >
+          <FormattedMessage
+            id="xpack.securitySolution.guided_onboarding.nextStep.buttonLabel"
+            defaultMessage="Next"
+          />
+        </EuiButton>
+      </EuiFlexItem>
+    </EuiFlexGroup>
+  );
+  const lastStepFooter = (
+    <EuiButtonEmpty
+      size="xs"
+      color="text"
+      onClick={() => resetTour()}
+      data-test-subj="onboarding--securityTourEndButton"
+    >
+      <FormattedMessage
+        id="xpack.securitySolution.guided_onboarding.endTour.buttonLabel"
+        defaultMessage="End tour"
+      />
+    </EuiButtonEmpty>
+  );
+  return tourConfig.map((stepConfig) => {
+    const { content, imageConfig, ...rest } = stepConfig;
+    return (
+      // @ts-expect-error
```

Comment:
Can you explain why this is needed?

Reply:
Good catch, @alisonelizabeth, I removed the error and fixed the config in this [commit](https://github.com/elastic/kibana/pull/132641/commits/763249e87a8a9bcb7c26a8070b7a5d7437e71814).